modules/ROOT/pages/export-to-pdf-standalone-service.adoc

```diff
@@ -1,5 +1,118 @@
+           "options": {
+             "output": "pdf"
+           }
+         }' --output ~/Downloads/html_with_inline_css.pdf
+----
+
+[[css]]
+=== CSS
+
+This option is used to send custom CSS styles to the converter.
+
+.Example of a simple CSS document to be converted to PDF
+[source,sh]
+----
+curl -X POST https://exportpdf.converter.tiny.cloud/v1/convert \
+     -H "Content-Type: application/json" \
+     -d '{
+           "html": "<!DOCTYPE html><html><head><title>Simple HTML Example PDF</title></head><body><h1>Simple HTML Example</h1><p>This is an example of a paragraph.</p><h2>Table Example</h2><table><thead><tr><th>Header 1</th><th>Header 2</th><th>Header 3</th></tr></thead><tbody><tr><td>Row 1, Cell 1</td><td>Row 1, Cell 2</td><td>Row 1, Cell 3</td></tr><tr><td>Row 2, Cell 1</td><td>Row 2, Cell 2</td><td>Row 2, Cell 3</td></tr></tbody></table><div>&copy; 2024 Example Company. All rights reserved.</div></body></html>",
+           "css": "p { color: red; }",
+           "options": {
+             "output": "pdf"
+           }
+         }' --output ~/Downloads/html_using_custom_css.pdf
+----
+
+[TIP]
+Just like the HTML, CSS must be encoded with UTF-8. If UTF-8 cannot be used and support for other charsets is needed, link:https://www.tiny.cloud/contact/[contact us].
+
+[[options]]
+=== Options
+
+The PDF options make it possible to set a header, footer, margins, and page format of the output file. There is also the `wait_time` parameter to control the render time.
+
+Refer to the https://exportpdf.converter.tiny.cloud/docs#section/PDF-options[converter documentation^] for details on the syntax and usage of these options.
+
+[[images]]
+=== Images
+
+Images can be either inserted from a URL or Base64-encoded, using the src attribute of the `+<img>+` tag.
+
+[[webfonts]]
+=== Web Fonts
+
+To generate PDF documents, web fonts can be used like in a regular browser. Specify the URL to the font file or a font file converted to the Base64 format.
+
+[NOTE]
+Web fonts are not supported in both the header and footer body and styles. For header and footer, use one of the default fonts.
+
+Refer to the link:https://exportpdf.converter.tiny.cloud/docs#section/Web-Fonts[converter documentation^] for details and examples.
+
+== The Export to PDF Editor Plugin
+
+The xref:exportpdf.adoc[Export to PDF] plugin allows easy printing of the WYSIWYG editor content to a PDF file. When enabled, this feature sends the content of the editor together with the styles that are used to display it to the {productname} Cloud Services HTML to PDF converter service. The service then generates a PDF document that can be downloaded.
+
+Thanks to this plugin, it takes exactly one button click to get a PDF file with content formatted the same way as displayed in the {productname} WYSIWYG editor. It provides an easy solution for integrations that do not require further processing of editor data once it is edited.
```

Comment:
@ShiridiGandham I feel like this section could be removed, as its only promoting the actual plugin.

Reply:
removed